src/implementation/Client/GRPCClient/configuration.ts

```diff
@@ -51,6 +51,7 @@ export default class GRPCClientConfiguration implements IClientConfiguration {
       for (const [key, value] of Object.entries(metadataObj)) {
         metadata.add(key, value);
```

Comment:
We do not need to pass `metadata` anymore to the call since we are populating `msg.getMetadataMap()`, so we can cleanup the `metadata` variable (at both places).

Reply:
Thanks, I wasn't sure if the parameter was used for something that the message metadata wasn't used for.  Fixed!